luminoth/tools/cloud/gcloud.py

```diff
@@ -125,6 +139,21 @@ def cloud_service(credentials, service, version='v1'):
     return discovery.build(service, version, credentials=credentials)
 
 
+def validate_region(region, project_id, credentials):
+    cloudcompute = cloud_service(credentials, 'compute')
+
+    regionrequest = cloudcompute.regions().get(
+        region=region, project=project_id
+    )
+    try:
+        regionrequest.execute()
+    except HttpError:
+        click.echo(
+            'Error: Couldn\'t find region "{}" for project "{}".'.format(
+                region, project_id))
+        sys.exit(1)
```

Comment:
sys.exit in an internal function? At least it should start with _ to make sure nobody outside this module calls it.

Reply:
Ok, will change.